### ISSUE TITLE
guidance on keywords

### ISSUE THREAD

**Issue by ianhi** [matplotlib/mpl-third-party#72]
Are there any limits on what keywords can/should be?

**Comment by jklymak** [comment]
no, they are free-form.  I guess there is some idea that they may go towards search at some point, but they are not used currently.

**Comment by timhoffm** [comment]
I suggest to just start using them as you see fit. We can always consolidate later.

**Comment by ianhi** [comment]
> I suggest to just start using them as you see fit. We can always consolidate later.

Sounds good for me. Perhaps this advice should be added somewhere public lest others do as I did and hold back from using more on account of uncertainty.
